tests/media_filename_test.go

```diff
@@ -137,6 +137,25 @@ func TestMediaFilenames(t *testing.T) {
 				}
 			})
 
+			t.Run("Will serve safe media types with parameters as inline", func(t *testing.T) {
+				if runtime.Homeserver != runtime.Synapse {
+					// We need to check that this security behaviour is being correctly run in
+					// Synapse, but since this is not part of the Matrix spec we do not assume
+					// other homeservers are doing so.
+					t.Skip("Skipping test of Content-Disposition header requirements on non-Synapse homeserver")
```

Comment:
So I guess the reason you're not using `runtime.SkipIf` is because you want to _only_ run this test on synapse, rather than skip if it is dendrite/conduit?

Reply:
It is what the other tests in this file used and I copied & pasted the test to bootstrap. 😄